src/components/SearchBox.js

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { IoSearch } from "react-icons/io5";
+import axios from "axios";
+
+export default function SearchBox({setUsers}){
+    const [searchName,setSearchName] = useState('');
+
+    function searchUser(event){
+        event.preventDefault();
+
+        const body = {
+            name:searchName
+        }
+    
+        const promise = axios.post('http://localhost:4000/timeline/user',body);
```

Comment:
switch the url for the deployed application

Reply:
Done